pkg/JuliaInterface/src/JuliaInterface.c

```diff
@@ -319,22 +319,21 @@ static Obj IsOutputStream;
 
 static Obj FuncSTREAM_CALL(Obj self, Obj stream, Obj func, Obj obj)
 {
-    syJmp_buf readJmpError;
-
     BEGIN_GAP_SYNC();
 
     if (CALL_1ARGS(IsOutputStream, stream) != True) {
         ErrorQuit("STREAM_CALL: <outstream> must be an output stream", 0, 0);
     }
-    if (!OpenOutputStream(stream)) {
+    TypOutputFile output = { 0 };
+    if (!OpenOutputStream(&output, stream)) {
```

Comment:
This is really brittle; we should either move `STREAM_CALL` to GAP, or else add a libgap-api that does the brunt of its work for us